src/main/java/bio/terra/service/snapshotbuilder/utils/HierarchyQueryBuilder.java

```diff
@@ -31,19 +34,21 @@ public class HierarchyQueryBuilder {
   static final String ANCESTOR_CONCEPT_ID = "ancestor_concept_id";
   static final String DESCENDANT_CONCEPT_ID = "descendant_concept_id";
 
+  private static final String PERSON_ID = "person_id";
+
   /**
    * Generate a query to find all parent concepts of a given concept, and for each parent to find
    * all its children.
    *
    * <pre>{@code
-   * SELECT cr.concept_id_1 AS parent_id, cr.concept_id_2 AS concept_id, child.concept_name, child.concept_code
+   * SELECT cr.concept_id_1 AS parent_id, cr.concept_id_2 AS concept_id, child.concept_name, child.concept_code, has_children, count
    *   FROM concept_relationship cr, concept AS child, concept AS parent
    *   WHERE cr.concept_id_1 IN (:all parents of conceptId:) AND cr.relationship_id = 'Subsumes'
    *   AND parent.concept_id = cr.concept_id_1 AND child.concept_id = cr.concept_id_2
    *   AND parent.standard_concept = 'S' AND child.standard_concept = 'S'
    * }</pre>
    */
```

Comment:
Do we want to remove sql comments and instead have a more durable comment? and use tests to document the sql?

Reply:
Since there's a ticket in the backlog for that (https://broadworkbench.atlassian.net/browse/DC-1000) I think it's better to leave it as-is for now.